doc/gb-model/implementation.rst

```diff
@@ -0,0 +1,326 @@
+..
+  SPDX-FileCopyrightText: Contributors to gb-open-market-model <https://github.com/open-energy-transition/gb-open-market-model>
+
+  SPDX-License-Identifier: CC-BY-4.0
+
+.. _implementation:
+
+##########################################
+Technical Implementation
+##########################################
+
+This section provides technical details about GB-specific implementation features.
+
+.. _powerplants-data:
+
+===============================================
+FES Powerplants Data
+===============================================
+
+Overview
+========
+
+The GB model uses powerplant capacity data from the Future Energy Scenarios (FES) workbook, enriched with technical and cost parameters to create a complete dataset ready for PyPSA network composition. This data replaces the default PyPSA-Eur powerplants dataset with GB-specific capacity projections.
+
+Data Pipeline
+=============
+
+The powerplants data flows through two main stages:
+
+1. **Capacity Aggregation** (``create_powerplants_table.py``)
+
+   - Processes FES workbook data (GB regions)
+   - Processes European supply data (neighboring countries)
+   - Maps technology names to PyPSA carriers
+   - Aggregates capacities by bus, year, carrier, and set
+
+2. **Cost Enrichment** (``create_powerplants_table.py``)
+
+   - Joins technology cost data (efficiency, VOM, fuel costs, etc.)
+   - Calculates marginal costs
+   - Fills missing values with sensible defaults
+   - Creates unique generator indices
```

Comment:
delete - none of this is done in that file.

Reply:
it is, though a bit rudimentary i guess. see `assign_technical_and_costs_defaults` in `create_powerplants_table`